prisma/schema.prisma

```diff
@@ -233,3 +247,21 @@ enum TaskStatus {
   DELETED
   COMPLETED
 }
+
+enum Seniority {
+  JUNIOR
+  INTERMEDIATE
+  SENIOR
+  MANAGER
+  VPE
+  DIRECTOR
+}
+
+enum CompanySize {
+  SEED_10
```

Comment:
This seems like a combination of funding round and company size. I'm not super familiar with startup funding, but I didn't think that funding round _restricted_ company size. Would it be more appropriate to have separate `FundingRound` enum, which would be Seed/SeriesA/etc., and `CompanySize` enum, which would just be a range of employee count?

Reply:
the funding is tied to the company names. These are based on the company size on our form on the website https://docs.google.com/forms/d/e/1FAIpQLSdC-AbV2SNhMS99yhY6r4Gu34UC9DhVWIM7utII6irV1pyaSw/viewform

Reply:
Maybe we should update this to something like:
```
enum CompanySize {
    Less_Than_10,
    10_to_30,
    30_To_100,
    ...,
}
```